doc/man1/s_server.pod

```diff
@@ -125,14 +130,34 @@ manual page.
 
 Print out a usage message.
 
-=item B<-accept port>
+=item B<-port port>
 
 The TCP port to listen on for connections. If not specified 4433 is used.
 
+=item B<-accept val>
+
+The TCP optional ost and port to listen on for connections. If not specified 4433 is used.
+
 =item B<-naccept count>
 
 The server will exit after receiving B<number> connections, default unlimited.
 
+=item B<-unix val>
+
+Unix domain socket to accept on.
+
+=item B<-unlink>
+
+For -unix, unlink existing socket first.
```

Comment:
I agree with all your comments. I have just taken the text from the -help option of each command. My english isn't good enought to supply better sentences.
:-(